Post/tests/test_views_posts.py

```diff
@@ -115,3 +115,26 @@ def test_delete_post_GET(self, client):
         )
 
         assertTemplateUsed(url, 'Post/deletePost.html')
+
+    @pytest.mark.django_db
+    def test_update_post_GET(self, client):
+
+        user = User.objects.create_user(
+            username='Amit', email='Test24@gmail.com', password='password2244'
+        )
```

Comment:
why not using a fixture?

Reply:
Hi @guy9050,
I tried to use the fixtures I made, and all the tests passed.
But I got a flake8 error that says that these fixtures are imported but not used.

Reply:
def test_update_post_GET(self, client, #fixtures_here):

Reply:
> def test_update_post_GET(self, client, #fixtures_here):

I did it, but I still get the same error.

Reply:
can you please copy here what you did or better push it here? that's not supposed to happen if you use the fixture :)

Reply:
So each fixture leave in a different scope of file 
see this sample :)
You should 
I could recommend maintaining tests/fixture

https://gist.github.com/peterhurford/09f7dcda0ab04b95c026c60fa49c2a68

But not to make i hard use the follwoing:
+from Post.tests import create_post
+from Post.tests import create_user

+    @pytest.mark.usefixtures("create_post")
+    @pytest.mark.usefixtures("create_user")
+    def test_update_post_GET(self, client, create_user, create_post):

+        user = create_user( username='Dvir', email='Dvir@gmail.com', password='password777')

+        post = create_post(user=user, nameOfLocation='Israel', photoURL='www.test.com', Description='cool place' )

Reply:
Yes! I will push it here.